bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -144,6 +145,26 @@ private boolean isAppInForeground() {
         return mActivityCount > 0 || mIsConfigChange;
     }
 
+    /**
+     * If it's a fresh start or back to main activity, we can safely clear cache
+     */
+    private boolean isFreshStartOrBackToMain(@NonNull Activity activity, Bundle savedInstanceState) {
+        if (savedInstanceState != null) {
+            return false;
+        }
+        ActivityManager systemService = (ActivityManager) activity.getSystemService(Context.ACTIVITY_SERVICE);
+        if (systemService == null) {
```

Comment:
It's not possible for `ActivityManager` to be `null` here so let's remove this check.